### PR TITLE
add convenient StringUtil isEmpty method

### PR DESCRIPTION
**Pull request by olamy**

### CODE REVIEW

File: jetty-util/src/main/java/org/eclipse/jetty/util/StringUtil.java [hunk 1]
```diff
@@ -470,7 +470,29 @@ public static boolean isBlank(String str)
...
+     *   isEmpty("a")    == false
+     *   isEmpty(".")    == false
+     *   isEmpty(";\n")  == false
+     * </pre>
+     *
+     * @param str
+     *            the string to test.
+     * @return true if string is null or only whitespace characters, false if non-whitespace characters encountered.
+     */
+    public static boolean isEmpty(String str){
```
Comment (gregw) [inline jetty-util/src/main/java/org/eclipse/jetty/util/StringUtil.java]: probably `isNullOrEmpty` is a better name... other than that LGTM

Comment (joakime) [reply jetty-util/src/main/java/org/eclipse/jetty/util/StringUtil.java]: I like `isEmpty()` better.

Comment (gregw) [reply jetty-util/src/main/java/org/eclipse/jetty/util/StringUtil.java]: I can live with that.....  but I will NEVER accept the curly in that position :)

Comment (olamy) [reply jetty-util/src/main/java/org/eclipse/jetty/util/StringUtil.java]: formatting fixed

**Pull request merged**